modules/phase_field/src/kernels/ACGrGrMulti.C

```diff
@@ -21,14 +21,16 @@ validParams<ACGrGrMulti>()
       "gamma_names",
       "List of gamma material property names for each other order parameter. Place "
       "in same order as order parameters (v)!");
+  params.addParam<MaterialPropertyName>("en_barr", "mu", "Energy Barrier Coefficient");
```

Comment:
In my model I found that interface widths are more consistent if a separate mu is defined for the grain boundaries and surfaces. Currently, the same mu is automatically used for all cases, so I added this input to let you use multiple mu's.

Once I did that though, the only way I could get the code to compile was to reinitialize it in the header. If there is a way to keep this input while initializing in `ACGrGrBase`, please let me know.

Reply:
Hm... this model is set up to use a consistent `mu` throughout. Are you saying you are using different values of `mu` in different kernels? If you do that, I'm afraid your evolution equations are no longer variationally derived from the free energy functional, so they do not guarantee decreasing energy.

Using large/small values of gamma changes interface width. Moelans addressed this in Acta Mater. 59, 1077-1086 (2011), and proposed to make interface widths more consistent by making kappa a function of order parameters. I am not a proponent of that approach because the evolution equations are again non-variational, but I think the governing equations in that model could be fixed by including the order parameter dependence in the kappas. Similarly, you could make mu a function of the order parameters and take the variational derivative of mu accounting for its kappa dependence.

Sorry I know this is opening a can of worms you probably didn't intend to with this PR, but unless we can implement this strategy I think we should keep mu to be a single value throughout currently.

Reply:
The interfacial energy in general is controlled by mu, kappa, and gamma- if you've seen the manuscript I wrote on the model, see eq. (43) (I call mu m there to be consistent with the 2011 Moelans paper - if you don't have the manuscript let me know and I can send it to you.) So to set interfacial energies, we keep mu and kappa constant throughout, and vary gamma. So we would want to change gamma to get different energies between different types of interfaces. Does that make sense?